app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -28,15 +30,24 @@ class Uploader extends \Magento\Backend\Block\Widget
     protected $_fileSizeService;
 
     /**
+     * @var \Magento\Framework\Json\EncoderInterface
+     */
+    private $jsonEncoder;
+
+    /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Framework\File\Size $fileSize
      * @param array $data
+     * @param \Magento\Framework\Json\EncoderInterface $jsonEncoder
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Framework\File\Size $fileSize,
-        array $data = []
+        array $data = [],
+        \Magento\Framework\Json\EncoderInterface $jsonEncoder = null
```

Comment:
This interface is deprecated, please check https://github.com/magento/magento2/pull/10342/files#r129685891 for an up-to-date way to inject JSON encoder.

Reply:
Good to know it and the reason why 👍